README.md

```diff
@@ -21,7 +23,18 @@ Please note: It might take a few hours until a new version is available on cdnjs
 
 ## Development
 
-Once you cloned the repo, you'll need to install [grunt](http://gruntjs.com/) and the development dependencies using [yarn](https://yarnpkg.com/).
+Once you cloned the repo, you'll need to install [grunt](https://gruntjs.com/) and the development dependencies using package manager:
 
-    $ [sudo] yarn global add grunt-cli
-    $ yarn install
+* [yarn](https://yarnpkg.com/):
```

Comment:
Add `(Recommended)`  at the end of this line. As we want to encourage people to use yarn. As it is faster, safer to use